setup.py

```diff
@@ -57,6 +57,7 @@
     install_requires=[
         'pygpgme>=0.3',
         'psycopg2',
+        'pyxarf',
```

Comment:
Right now the pypi verison of pyxarg 0.5.0 will probably not work, but the master version would.
In order to get a way to specify the dependency more properly, I've suggested to "release" a new version of pyxarf:
https://github.com/xarf/python-xarf/pull/2

Thanks @wagner-certat for pointing out that this dependeny is missing.

Reply:
Specifying the direct git/github url does not work inside the setup.py parameters. I tried several possibilities, but nothing worked. So it needs to be documented "manually".

Reply:
Added manual documentation with https://github.com/Intevation/intelmq-mailgen/commit/7957676c83d32e40daa31b53ec6e83a34496816a Thanks @wagner-certat"